api/server/backup.go

```diff
@@ -302,13 +332,26 @@ func (vd *volAPI) cloudBackupStateChange(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
-	d, err := vd.getVolDriver(r)
+	// Get context with auth token
+	ctx, err := vd.annotateContext(r)
 	if err != nil {
-		notFound(w, r)
+		vd.sendError(vd.name, method, w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	// Get gRPC connection
+	conn, err := vd.getConn()
+	if err != nil {
+		vd.sendError(vd.name, method, w, err.Error(), http.StatusInternalServerError)
 		return
 	}
 
-	err = d.CloudBackupStateChange(stateChangeReq)
+	volumes := api.NewOpenStorageCloudBackupClient(conn)
```

Comment:
nit: this is not the volumes api, it's the cloudbackups api.

Reply:
Done.